src/js/modules/forms/components/SectionForm.js

```diff
@@ -17,20 +25,28 @@ const styles = {
     padding: "5%",
 
   },
+  inputs: {
+    paddingLeft: "2%"
+  },
   titleInput: {
-    padding: "2%"
+    paddingBottom: "2%"
```

Comment:
These all should be margins, not paddings. Not only is margin more intuitive (margin is outside the element and its border whereas padding is inside), vertical margins auto-collapse. That means that you can do `margin: 2% 0` for all three inputs and the margin won't add up to 4%—it'll be 2%.

Also, why are you using percentages? Just use constants.